### DISCUSSION TITLE
Arms warrior "Deep wounds" pandemic window

### DISCUSSION THREAD

**Discussion by Fosphen** [simulationcraft/simc#5825] [General]
Hi there, I would like to know if [Mortal Strike]'s "Deep Wounds" pandemic window is implemented and working in SimC.
Thank you :)

**Comment by scamille** [comment]
cc @simulationcraft/warrior

**Reply by Fosphen** [reply]
Ty for the answer but I don't see any Simulationcraft/warrior repository and your link leads to error 404.

**Reply by scamille** [reply]
I just wanted to ping to warrior team in simulationcraft, so they might give you a more appropriate answer. Not sure if that works in discussions...

**Comment by Fosphen** [comment]
:Cough of a lonely man echoing in a cold and silent cave:
